### PR TITLE
Document new Microsoft Black Formatter extension for VSCode

### PR DESCRIPTION
**Pull request by laundmo**
### Description

VSCode announced a [new extension](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter) for Black in their update notes for version 1.67.0, this PR adds a sentence to the editor integrations docs about this extension.

### Checklist - did you ...

- [ ] Add a CHANGELOG entry if necessary?
- [ ] Add / update tests if necessary?
- [x] Add new / update outdated documentation?

### CODE REVIEW

File: docs/integrations/editors.md [hunk 1]
```diff
@@ -301,9 +301,10 @@ close and reopen your File, _Black_ will be done with its job.
 
 ## Visual Studio Code
 
-Use the
+- Use the
```
Comment (cooperlees) [inline docs/integrations/editors.md:304]: Why was this indented? I think this is what is making linting upset.

**Review by cooperlees — CHANGES_REQUESTED** [review]
Seems helpful for readers. Please just fix the lint.

**Review by ichard26 — APPROVED** [review]
The VSCode team is planning to remove the formatting integration with Black in the main Python extension eventually, but we can deal with that later when it happens :)

Nice to see you around @laundmo. Hope you've been doing well! Thank you for the PR!

**Comment by ichard26** [comment]
Congrats on your first PR to psf/black @laundmo! :tada:

**Pull request merged**